### PR TITLE
Hotfix application flag change to only FLAG_DEBUGGABLE by &= operator

### PR DESCRIPTION
**Pull request by Toxa2033**
Hotfix application flag change. It may affected on many aspect of application. In my case some layout behaved unpredictably because vwo set flag only FLAG_DEBUGGABLE or zero when check isApplicationDebuggable

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 android.enableD8=true
 
-VERSION_NAME=2.8.0
-VERSION_CODE=27
+VERSION_NAME=2.8.1
```
Comment (softvar) [inline gradle.properties:3]: can you please explain the reasons for changing this?

File: library/src/main/java/com/vwo/mobile/utils/VWOUtils.java [hunk 2]
```diff
@@ -179,7 +179,7 @@ private static String capitalize(String str) {
      * @return {@link Boolean#TRUE} is app is running in debug mode else {@link Boolean#FALSE}
      */
     public static boolean isApplicationDebuggable(@NonNull Context context) {
-        return (0 != (context.getApplicationInfo().flags &= ApplicationInfo.FLAG_DEBUGGABLE));
+        return (0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE));
```
Comment (softvar) [inline library/src/main/java/com/vwo/mobile/utils/VWOUtils.java:182]: What error did you encounter while using previews?

File: gradle.properties (hunk 1 above)
Comment (Toxa2033) [reply gradle.properties:3]: Just for update version

File: library/src/main/java/com/vwo/mobile/utils/VWOUtils.java (hunk 2 above)
Comment (Toxa2033) [reply library/src/main/java/com/vwo/mobile/utils/VWOUtils.java:182]: if we expand the operator &= we get `context.getApplicationInfo().flags = context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE`.  Its means it method change context.getApplicationInfo().flags. This flag has many settings inside, like FLAG_SUPPORTS_SCREEN_DENSITIES and other screen support flags. And when we use isApplicationDebuggable all flags in getApplicationInfo().flags become disabled. This leads to the fact that the layout is destroyed in some places

Comment (softvar) [reply library/src/main/java/com/vwo/mobile/utils/VWOUtils.java:182]: Got it...thanks for the explanation 👍

**Review by softvar — APPROVED** [review]

**Comment by softvar** [comment]
This is released in 2.8.2

**Pull request merged**